src/Javis.jl

```diff
@@ -1084,6 +1090,10 @@ function javis(
         end
         filecounter += 1
     end
+    
+    if liveview == true
+        javis_viewer(video = video, frames = length(frames), action_list = actions)
+    end
```

Comment:
I as a user expect that the live viewer shows up immediately and that no frames are getting rendered.
So `liveview=true` means: Showing the live viewer without rendering as we want to have a fast experience which was the main point for having an image viewer in the first place :wink:

Reply:
This makes a lot of sense! I fixed this correctly now. :D